fdbt-dev/Makefile

```diff
@@ -178,6 +178,10 @@ validate-netex:
 validate-latest-file:
 	./scripts/validate_latest_netex.sh
 
+# usage is 'make file=<path-to-file> validate-this-netex'
+validate-this-netex:
+	./scripts/validate_this_netex.sh $(file)
```

Comment:
is it similar to validate-netex in the makefile

Reply:
that one doesnt work, for some unknown reason.